sql/deployments/deployment-2018-01-16-COLAB-2495.sql

```diff
@@ -0,0 +1,18 @@
+# Display all entries that have the issue
+SELECT * FROM xcolab.members_Member WHERE members_Member.status IS  NULL ;
+
+# Display entries without a createDate and the replacement createDate (DryRun)
+SELECT T1.id_, T1.screenName, T1.createDate, T2.createDate
+FROM xcolab.members_Member T1, xcolab.members_Member T2
+WHERE T1.id_ = T2.id_ + 1 AND T1.createDate IS NULL;
+
+# Update status fields
+UPDATE xcolab.members_Member SET status=0 WHERE status IS NULL;
+UPDATE xcolab.members_Member SET autoRegisteredMemberStatus=0 WHERE autoRegisteredMemberStatus IS NULL;
```

Comment:
Please remove the database name (`xcolab.`) from the queries, as they may not be the same for everyone.